_posts/2023-03-15-new-release-2023-1.md

```diff
@@ -0,0 +1,9 @@
+---
+title:  "Connector SDK now targets Tableau 2023.1"
+abstract: "The master SDK branch now targets Tableau 2023.1. The Connector SDK for 2022.4 has released for those wishing to target that version."
+---
+
+On March 15, with the full release of Tableau 2023.1 we moved the latest version of the master branch to target Tableau 2023.1. This means that the samples and connectors made with this version of the SDK may not work with older versions of Tableau. To develop against Tableau 2022.4, please download the [Tableau Connector SDK for Tableau 2022.4](https://github.com/tableau/connector-plugin-sdk/releases/tag/tableau-2022.4) release.
+
+__Support for multiple OAuth configurations__ <br>
+More details... See [OAuth Authentication Support](https://tableau.github.io/connector-plugin-sdk/docs/oauth) page for additional details.
```

Comment:
'''suggestion
Once a connector has been updated, this release includes a major platform update that allows a Tableau Administrator to fully configure the OAuth flow for their IDP of choice. For more information about how to update your connector, see: [OAuth Authentication Support](https://tableau.github.io/connector-plugin-sdk/docs/oauth) page for additional details.
'''